src/index.html

```diff
@@ -1,12 +1,444 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+        >
+        <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__images"
+          >
+
+        <h2 class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+            Product code: 195434
+        </p>
+
+        <div class="card__stars-two">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-text">
+              Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+              Price:
+          </p>
+
+          <p class="card__price-sum">
+              $2,199
+          </p>
+        </div>
+
+        <a
+            href="#"
+            data-qa="card-hover"
+            class="card__button"
+          >
+            Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__images"
+          >
+
+        <h2 class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+            Product code: 195434
+        </p>
+
+        <div class="card__stars-two">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-text">
+              Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+              Price:
+          </p>
+
+          <p class="card__price-sum">
+              $2,199
+          </p>
+        </div>
+
+        <a
+            href="#"
+            class="card__button"
+          >
+            Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__images"
+          >
+
+        <h2 class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+            Product code: 195434
+        </p>
```

Comment:
It will be better if you use 2 spaces for nested tags everywhere, now in one place 4 spaces, in some 2 spaces. It is necessary to adhere to a single principle
```suggestion
        <p class="card__code">
          Product code: 195434
        </p>
```